netbout-web/src/test/java/com/netbout/rest/TkAppITCase.java

```diff
@@ -57,6 +58,18 @@
         Manifests.read("Netbout-Documentation");
 
     /**
+     * Xpath to the 'page not found' notice.
+     */
+    private static final String MISSING =
```

Comment:
@mkordas Yes, sorry I was a little unsure whether or not do inline it here. I would have inlined it, but given that you asked me to setup a constant for a long regex I assumed we had some rule about long strings here :)
Should I inline it ?